buildSrc/src/main/java/Config.kt

```diff
@@ -133,6 +133,7 @@ object Config {
         val apolloKotlin = "com.apollographql.apollo3:apollo-runtime:3.3.0"
 
         val otelVersion = "1.19.0"
+        val otelAlphaVersion = "1.19.0-alpha"
```

Comment:
I think we usually don't use preview versions for production code dependencies, dunno if it's fine in this case

Reply:
Some dependencies are only released as alpha at the moment. So it's not a matter of us wanting to use a newer version but them releasing some as final some as alpha in the same release. See https://github.com/open-telemetry/opentelemetry-java/blob/main/VERSIONING.md

Imo it's fine to get us started and collect some feedback.